CMakeGraphVizOptions.cmake

```diff
@@ -0,0 +1,7 @@
+set(GRAPHVIZ_GENERATE_PER_TARGET FALSE)
```

Comment:
I don't believe so. The file needs to be in `CMAKE_BINARY_DIR` or `CMAKE_SOURCE_DIR` so it can be automatically loaded when CMake is run with the `--graphviz` option,